src/content/pages/accessibility.mdx

```diff
@@ -0,0 +1,257 @@
+
+## Sprints / Unconference Venue (Sat & Sun)
+
+The sprints and unconference will take place at a [different venue](/venue#where-1) than the main conference.
+{/*
+
+The **Vysoká škola ekonomická v Praze - Italská budova** (Prague University of Economics and Business, Žižkov Campus) is  accessible for wheelchair users and other people with limited mobility.
+
+### Parking
+Parking at the venue is not available to us. We recommend using public transport to get there. If you are handicapped and need parking, please contact us at  helpdesk@europython.eu and we will try to arrange parking for you.
+
+### Entrance
+The entrance to the building is on **Italská Street**.  It is wheelchair accessible via a ramp to the left of the main entrance. You do not need to use the stairs to enter the building.
+### Inside the building
+The building is accessible for wheelchair users and people with limited mobility. The interior is designed to accommodate various accessibility needs.
+
+The sprints and lunch / coffee breaks will happen on the ground floor at the end of the hallway, to get there, you can use the stairs or the lift located in the middle of the building.
+
+Unconference rooms are located on the first floor, and you can access them via the stairs or the lift located in the middle of the building.
+
+### Childcare
+Unfortunately, we will not be able to provide childcare at the sprints venue.
+
+*/}
+
+
+
+## Accessibility FAQ
+
+This FAQ covers accessibility information for EuroPython. The EuroPython team
+recognises that accessibility is a continuous process, both systemic and
+individual.
+
+If you have specific questions or accessibility needs not covered here, please
+email us at **helpdesk@europython.eu**.
+
+<Accordion title="Allergies" id="allergies">
+
+Refrain from wearing scents such as perfumes/colognes, scented lotions,
+clothing with strong detergent scents, etc. Notify us of preventable
+environmental allergies when purchasing tickets or by email.
+</Accordion>
+
+
+<Accordion title="Will there be captioning?" id="captioning">
+
+Unfortunately we do not have captioning during EuroPython.
+</Accordion>
+
+
+<Accordion title="What about interpreters?" id="interpreters">
+
+  We do not plan to have interpreters at the conference unless we are notified
+  by signing attendees. If you have questions, please email us.
+</Accordion>
+
+<Accordion title="What about hearing aid induction loops?" id="induction-loops">
+ We do not have hearing aid induction loops at the conference. But all speakers are using microphones, and we live stream the talks (Wed-Fri) with audio, so you can listen to the talks from your own device. Unfortunately these streams do have a slight delay.
+</Accordion>
+
+<Accordion title="Any suggestions/tips as a volunteer MC or Session Chair?" id="volunteer-tips">
+
+- Session chairs/room volunteer(s): Let folks know if there's loud, unexpected
+  noises, strobing/flashing visuals in speakers' presentations beforehand,
```

Comment:
```suggestion
- Session chairs/room volunteer(s): Let folks know in advance about any loud or unexpected
  noises, strobing/flashing visuals in speakers' presentations, or
```

Reply:
I'd also consider dropping the Session chairs / room volunteers. Just start with "Let folks know". Because it's repetitive.